### PR TITLE
typo in vignette and package imports

### PR DESCRIPTION
**Pull request by kevinrue**

### CODE REVIEW

File: R/iSEE-main.R [hunk 1]
```diff
@@ -946,7 +946,7 @@ iSEE <- function(
         }
     }
 
-    # Linked selection structure observers for the heatmaps.
+    # Brush structure observers for the heatmaps.
```
Comment (LTLA) [inline R/iSEE-main.R]: This is really a zoom observer, nothing to do with brushing here.

Comment (kevinrue) [reply R/iSEE-main.R]: Indeed, now that you pointed it out, I noticed the .zoomClick under the .brushField.
Fixed. Thanks!

**Comment by kevinrue** [comment]
Shall we merge or up the code coverage?

**Comment by LTLA** [comment]
It's good enough.

**Pull request merged**